astropy/timeseries/binned.py

```diff
@@ -11,6 +12,7 @@
 
 from astropy.timeseries.core import BaseTimeSeries, autocheck_required_columns
 
+
```

Comment:
You seem to have added an extra line here - try to not change things that are not related to the actual code change. It might help to know coding style conventions (PEP8 conventions) mentioned here: https://docs.astropy.org/en/stable/development/codeguide.html in the future. It is hard to learn at first (and seems nitpicky) but that's how `astropy` keeps its code well-maintained!